terminal/cursor.go

```diff
@@ -47,18 +47,24 @@ func (c *Cursor) Back(n int) error {
 
 // NextLine moves cursor to beginning of the line n lines down.
 func (c *Cursor) NextLine(n int) error {
-	if err := c.Down(1); err != nil {
+	if err := c.HorizontalAbsolute(0); err != nil {
```

Comment:
Why first move horizontal and then try to move between lines?

Reply:
There seems to be weird behavior when going `Down(0)` or `Up(0)`, where the cursor moves down or up `1` instead of remaining on the same line. I'm not sure if this is [expected ANSI behavior](https://en.wikipedia.org/wiki/ANSI_escape_code#CSI_(Control_Sequence_Introducer)_sequences) for `\x1b[0A`, but this change was made to guard against this case.

It might be more appropriate to move this check for `n == 0` to the `Down` and `Up` functions instead? I wasn't sure if this current behavior was relied on by other functions and felt that this change was safer.